controllers/usersController.js

```diff
@@ -48,6 +48,29 @@ const getUser = async (req, res) => {
   }
 }
 
+/**
+ * Checks whether a user exists or not
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const checkUser = async (req, res) => {
+  try {
+    const result = await userQuery.fetchUser({ username: req.params.username })
+    if (!result.userExists) {
+      return res.json({
+        message: 'User name Available',
+        userExists: result.userExists
+      })
+    }
+    return res.boom.notFound('User name not Available')
```

Comment:
Any update on this? As per discussed in the meeting, we were to send a 200 with userExists as false.